frontend/__tests__/routes/_protected/applications/index.test.tsx

```diff
@@ -112,7 +112,7 @@ vi.mock('~/services/benefit-application-service.server', () => ({
               socialInsuranceNumber: '723 435 814',
             },
             communicationPreferences: {
-              email: 'communicat@bowl.ca',
+              email: 'firstlady@bowl.ca',
```

Comment:
I don't think you should use a potentially-valid email address here.

If this is indeed someone's actual email address, we risk accidentally spamming this person. Instead, use `@example.com` email addresses.

Reply:
Fixed: 694baf21af9cf7888aa15e5cfc53d881b3910e9f